codespell_lib/data/dictionary.txt

```diff
@@ -30861,6 +30988,7 @@ intereferes->interferes
 interefering->interfering
 interelated->interrelated
 interelaved->interleaved
+interent->internet, interest, inherent,
```

Comment:
Also `interment` and `interlent` (past of `interlend`)?

Reply:
I'll add `interment`. I can't find typos of `interlent`.